### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DipanshKhandelwal**
Added setup instructions in README.md . #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,4 +4,79 @@ View online at [iiitv.github.io](http://iiitv.github.io).
...
+ ### 4. Running the project
+ * Assuming your terminal is at your repository.
+ * If not follow step 2.
+ ##### To run the project
+ * Do the following
+ ```
+ jekyll serve
+ ```
+ * This command will run the project on default ``` localhost:4000 ```.
+ * Open any browser and got to the address ``` localhost:4000 ```.
```
Comment (aviaryan) [inline README.md]: *got

File: README.md [hunk 2]
```diff
@@ -4,4 +4,80 @@ View online at [iiitv.github.io](http://iiitv.github.io).
...
+ ###### Do the following
+ ```
+ gem install jekyll
+ ```
+ 
+ #### You also need to install Redcarpet
+ * To install redcarpet do the following
+ ```
+ sudo gem install redcarpet
+
```
Comment (aviaryan) [inline README.md]: Please clean the markdown source file i.e. remove extra blank lines and add blank lines where necessary. (example after a `####` line)

Comment (DipanshKhandelwal) [reply README.md]: Ok ill do it ASAP.

File: README.md [hunk 3]
```diff
@@ -4,4 +4,90 @@ View online at [iiitv.github.io](http://iiitv.github.io).
...
+ ```
+ gem install jekyll
+ ```
+ 
+ #### (iv) You also need to install Redcarpet
+ 
+ ###### To install redcarpet do the following
+ ```
+ sudo gem install redcarpet
+
```
Comment (anshumanv) [inline README.md]: Extra blank line ?

**Review by anshumanv — COMMENTED** [review]
Check comment

File: README.md (hunk 3 above)
Comment (DipanshKhandelwal) [reply README.md]: Thanks, it was left by mistake.

**Review by aviaryan — APPROVED** [review]

**Comment by aviaryan** [comment]
Thanks for the PR 😃. Looking forward to more contributions from you.

**Comment by DipanshKhandelwal** [comment]
Yes, ill try to contribute.

**Pull request merged**